### PR TITLE
21357 cleanup formatting typos and lints in sorted collection test and related traits

### PR DESCRIPTION
**Pull request by astares**
https://pharo.fogbugz.com/f/cases/21357

Looks a lot but it is primarily
- formatting 
- typo fixing
- comment fixing 
- and assert:equals conversion. 
- sometimes also a change in the iterator to simplify and satisfy critics by using #withIndexDo:

No change in behavior - is safe to include. Was done in recent Pharo-7.0+alpha.build.552

If one wants to review it would be best to use Pharo differ instead of git diff.

### CODE REVIEW

File: src/Collections-Tests/TAddTest.trait.st [hunk 1]
```diff
@@ -36,7 +37,7 @@ TAddTest >> test0FixtureRequirementsOfTAddTest [
 { #category : #'tests - adding' }
 TAddTest >> testTAdd [
 	| added collection |
-	collection :=self otherCollection .
+	collection :=self otherCollection.
```
Comment (jecisc) [inline src/Collections-Tests/TAddTest.trait.st]: If you want to improve the formating maybe you can also add a space after the assignation opperator.

File: src/Collections-Tests/TIncludesWithIdentityCheckTest.trait.st [hunk 2]
```diff
@@ -76,7 +78,8 @@ TIncludesWithIdentityCheckTest >> testIdentityIncludes [
 
 { #category : #'tests - includes' }
 TIncludesWithIdentityCheckTest >> testIdentityIncludesNonSpecificComportement [
-	" test the same comportement than 'includes: '  "
+	" test the same comportement than 'includes: ' "
```
Comment (jecisc) [inline src/Collections-Tests/TIncludesWithIdentityCheckTest.trait.st]: There is a missing uppercase here

File: src/Collections-Tests/TAddTest.trait.st (hunk 1 above)
Comment (astares) [reply src/Collections-Tests/TAddTest.trait.st]: I already did that on several places. Maybe I missed some - will do another pass.

**Review by jecisc — APPROVED** [review]
If tests pass this is good for me. I added 2 small comments but it's not important if they are not taken into account, this PR already improve a lot the situation.

**Comment by astares** [comment]
I completed more of them. Tests pass - all green.

**Pull request merged**